File: 30-days-of-code/Day02/Operators.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    Operators.py                                       :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/03/05 18:29:53 by alaamimi          #+#    #+#              #
#    Updated: 2021/03/05 18:33:14 by alaamimi         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

#!/bin/bash 

import math
import os
import random
import re
import sys

def solve(meal_cost, tip_percent, tax_percent):
    tip = tip_percent / 100 * meal_cost
    tax = tax_percent / 100 * meal_cost
    total_cost = round(tax + tip + meal_cost)
    print(total_cost)

if __name__ == '__main__'
    meal_cost = float(input())
    tip_percent = int(input())
    tax_percent = int(input())
    solve(meal_cost, tip_percent, tax_percent)

File: 30-days-of-code/Day02/Operators.java
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   Operators.java                                     :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/02/25 04:02:16 by alaamimi          #+#    #+#             */
/*   Updated: 2021/02/25 04:02:20 by alaamimi         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

import java.util.Scanner;
import java.lang.Math;

public class Operators
{
	public static void main(String[] args)
	{
		Scanner scanInput = new Scanner(System.in)

		double mealCost = scanInput.nextDouble(); //Original meal price
		int tipPercent = scanInput.nextInt(); //Tip percent
		int taxPercent = scanInput.nextInt(); //Tax percent

		scanInput.close(); //End of reding input from User

		//Calculations Code
		double tip = mealCost * tipPercent / 100; //
		double tax = mealCost * taxPercent / 100;

		//Typecast the result at the rounding operation of an int
		int totalCost = (int) Math.round(tax + tip + mealCost);

		//Print result
		System.out.println(totalCost);
	}
}
